circle.yml

```diff
@@ -40,3 +42,5 @@ deployment:
         --workdir /go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
         --entrypoint make
         segment/golang:latest release
+      - fpm -s dir -t deb -n chamber -v $(CIRCLE_TAG) ./dist/chamber--linux-amd64=/usr/bin/chamber
```

Comment:
Instead of putting these two lines in the circle yaml directly, they should be in the make file.  Then that make target should be invoked in the `deployment` section of the circle yaml so that we only build/push packages on tagged commits.  The deployment section already runs one make target, you should be able to just add an additional make target to that invocation.